recipes/spyrmsd/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spyrmsd" %}
+{% set version = 0.3.0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 985861283513fcf9979c028ccd4a9c8dfb7771673a77a8182104de618950e261
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
Do you need a compiler for this package? I don't see any C-extensions being built in the `setup.py`.